### DISCUSSION TITLE
[wayland] Virtual Pointer protocol

### DISCUSSION THREAD

**Discussion by Sydiepus** [qtile/qtile#3587] [Ideas] [answered]
wayvnc fails to start because Virtual Pointer protocol not supported by compositor.
it looks like pywayland does not even support it, i checked [here](https://pywayland.readthedocs.io/en/latest/module/protocol/index.html).
not sure what you can do if that's the case, but i'm not confident about my information.

**Comment by m-col** [comment, marked as answer]
Currently this protocol is not supported, but I've implemented support here so it will be out in the next release: https://github.com/qtile/qtile/pull/3589

**Reply by Sydiepus** [reply]
Thank you.
